Kickstarter-iOS/Views/PledgeCTAContainerView.swift

```diff
@@ -8,14 +8,27 @@ final class PledgeCTAContainerView: UIView {
 
   private let vm: PledgeCTAContainerViewViewModelType = PledgeCTAContainerViewViewModel()
 
-  private lazy var amountAndRewardTitleStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var amountAndRewardTitleStackView: UIStackView = {
```

Comment:
Enabling auto layout in initializers thorough this file so that we don't have to combine it with style